scripts/build_hydro_profile.py

```diff
@@ -207,9 +207,9 @@ def approximate_missing_eia_stats(eia_stats, runoff_fn, countries):
     norm_year = config_hydro.get("eia_norm_year")
     missing_years = contained_years.difference(eia_stats.index)
     if norm_year:
-        eia_stats.loc[contained_years] = eia_stats.loc[norm_year]
+        eia_stats.loc[contained_years[0]] = eia_stats.loc[norm_year]
```

Comment:
Why is this ?

Reply:
Ideally this should not have to be done. but because 2023 is not within the existing EIA hydro statistics, it keeps getting error unless I do this. 

Drawbacks: Not being able to model longer than 1 year. Suggestion is very appreciated!